index.js

```diff
@@ -130,7 +130,18 @@ module.exports = (app) => {
     let output = execSync("cd /tmp/vitess && go run ./go/vt/vterrors/vterrorsgen");
     let errStrVitess = output.toString()
 
-    const docPath = "/tmp/website/content/en/docs/15.0/reference/errors/query-serving.md"
+    const pr_details = await context.octokit.rest.pulls.get({
+      owner: pr.owner,
+      repo: pr.repo,
+      pull_number: pr.pull_number,
+    });
+
+    let currVersion = "16.0";
```

Comment:
note that this will need to be updated once every major release of Vitess

Reply:
Is there some way to infer this from the vitess repository? We did something similar for the upgrade-downgrade tests, didn't we?

Reply:
@GuptaManan100, there is a way to dynamically set this using the vitessio/website repository. We store a `next` variable in the `./config.toml` file that defines the current development version AKA the version of vitessio/vitess `main`. In 3c59f97 I am using this file and some bash commands to dynamically set the current version.